profile/README.md

```diff
@@ -0,0 +1,55 @@
+![Alephium Logo](https://github.com/alephium/alephium-brand-guide/blob/master/logos/grey/Logo-Horizontal-Grey.png "Alephium Logo")
```

Comment:
Let's use theme-aware logos like Hongchao? https://github.com/alephium/extension-wallet/blob/master/Readme.md?plain=1#L3

If we cannot use theme-aware logs, why do we expect the external services to use them?

Reply:
Wow! This is great! Fixed in https://github.com/alephium/.github/commit/79d5646f2f4db327c6bdb6718e2d2b87ae851656